lib/kafka_ex/protocol/metadata.ex

```diff
@@ -1,5 +1,6 @@
 defmodule KafkaEx.Protocol.Metadata do
   alias KafkaEx.Protocol
+  import KafkaEx.Protocol.Common
```

Comment:
I personally would not use `import` but I don't have a strong opinion against it